capstone-frontend/src/components/BookPage.jsx

```diff
@@ -79,7 +79,7 @@ export class BookPage extends Component {
                         starSpacing='10px'
                         starRatedColor='gold'
                       />
-                      {book.avgRating === 0 && <p id='rating-label'>No Ratings</p>}
+                      {book.avgRating < 1 && <p id='rating-label'>No Ratings</p>}
```

Comment:
Is it possible for a book to actually have an average rating less than 1 (i.e. can you give it 0 starts or is 1 the lowest)? If not this looks good.

Reply:
If someone gives a rating to a book on the Google Books site, 1 is the lowest possible rating so there can be no possible average rating below 1.

On the backend I return -1 in the case of no rating because I feel that makes it more clear that it is "invalid" whereas an `avgRating` of 0 could be seen as meaning every person who reviewed gave 0 stars, which is not the case.